opencti-platform/opencti-front/src/utils/graph/EntityDetails.tsx

```diff
@@ -142,6 +142,15 @@ const entityDetailsQuery = graphql`
         name
         x_mitre_id
         description
+        killChainPhases {
```

Comment:
Maybe, you can try to use the representative mechanism here.
Less complicated that the previous point (I think)

I see usage of defaultValue